napari/utils/action_manager.py

```diff
@@ -173,7 +174,7 @@ def _update_shortcut_bindings(self, name: str):
         if name not in self._shortcuts:
             return
         action = self._actions[name]
-        km_provider: KeymapProvider = action.keymapprovider
+        km_provider = action.keymapprovider
```

Comment:
I think this change can be reverted, right? Because this method shouldn't be called if keymapprovider is None (from like 164, 165)